### ISSUE TITLE
Add a button to switch the modal

### ISSUE THREAD

**Issue by MaximeHollebecq** [med-material/ArduinoReactionSynchTests#4]
acutally , if we want to switch the modal (LED , ERM) in the firmware , we have to modify it in the .ino code. 
![image](https://user-images.githubusercontent.com/23449138/72334956-73a37600-36be-11ea-887e-42857863d9db.png)

the idea is to add a button on the board that switch the boolean value and give information on which modality we are : 
- if LED modal : LED light up for 1 s
- if ERM modal : ERM vibrate for 1 s

**Comment by bastianilso** [comment]
I think the reason to we have not gone this road, is because it would be very "specific" to the needs of the reaction time / synch test exercise.. Ie. it is less useful for other cases. @hendrikknoche what do you think?

**Comment by hendrikknoche** [comment]
yes, this could be nice to have to reduce the number of ino files students have to install/download and if the Arduino code can handle making sure the data is correctly written to the database (reaction vs. synch).

This would be best if no further soldering is required i.e. the 2nd button (pre-installed) on the shield is already working and we can simply address it from the code. if we combined the reaction and synch time ino files we should default to reaction time. (in case there is no second button if the students set up breadboards by hand).
I'm not sure which other cases we have with which this could collide @bastianilso
